docs/quickstart/quickstart_chains/ethereum-uniswap.md

```diff
@@ -0,0 +1,264 @@
+- [TransactionHandlers](../../build/manifest/ethereum.md#mapping-handlers-and-filters): On each and every transaction that matches optional filter criteria, run a mapping function
+- [LogHanders](../../build/manifest/ethereum.md#mapping-handlers-and-filters): On each and every log that matches optional filter criteria, run a mapping function
+
+In this Uniswap project, the primary focus is on indexing logs exclusively from two Uniswap smart contracts. LogHandlers, which are the prevalent type of Ethereum handlers, are showcased in this example project, totaling 10 distinct log handlers.
+
+Additionally, it's important to highlight that this project incorporates six different ABIs. Furthermore, to accommodate scenarios where a contract factory generates new contract instances, [dynamic data sources](../../build/dynamicdatasources.md) are employed.
+
+```yaml
+dataSources:
+  - kind: ethereum/Runtime
+    startBlock: 12369621
+    options:
+      # Must be a key of assets
+      abi: Factory
+      address: "0x1F98431c8aD98523631AE4a59f267346ea31F984"
+    assets:
+      Factory:
+        file: "./abis/Factory.json"
+      ERC20:
+        file: "./abis/ERC20.json"
+      ERC20SymbolBytes:
+        file: "./abis/ERC20SymbolBytes.json"
+      ERC20NameBytes:
+        file: "./abis/ERC20NameBytes.json"
+      Pool:
+        file: "./abis/pool.json"
+    mapping: ...
+    # ethereum/contract
+  - kind: ethereum/Runtime
+    startBlock: 12369651
+    options:
+      abi: NonfungiblePositionManager
+      address: "0xC36442b4a4522E871399CD717aBDD847Ab11FE88"
+    assets:
+      NonfungiblePositionManager:
+        file: ./abis/NonfungiblePositionManager.json
+      Pool:
+        file: ./abis/pool.json
+      Factory:
+        file: ./abis/factory.json
+      ERC20:
+        file: ./abis/ERC20.json
+    mapping: ...
+templates:
+  - name: Pool
+    kind: ethereum/Runtime
+    options:
+      abi: Pool
+    assets:
+      Pool:
+        file: "./abis/pool.json"
+      Factory:
+        file: "./abis/factory.json"
+      ERC20:
+        file: "./abis/ERC20.json"
+    mapping: ...
+```
+
+Check out our [Manifest File](../../build/manifest/ethereum.md) documentation to get more information about the Project Manifest (`project.yaml`) file.
+
+## 2. Update Your GraphQL Schema File
+
+The `schema.graphql` file determines the shape of your data from SubQuery due to the mechanism of the GraphQL query language. Hence, updating the GraphQL Schema file is the perfect place to start. It allows you to define your end goal right at the start.
+
+Within this project, you will observe the presence of 22 GraphQL entities, each interconnected through numerous foreign key relationships. Consider, for instance, the relationship between the `Pool` and `Token` entities. Importantly, these relationships encompass not only one-to-many connections but also extend to encompass many-to-many associations.
```

Comment:
I would show Pool and Token entities in the graphql shell

Reply:
Also we can consider adding a link to Hero Course part dedicated to relationships https://academy.subquery.network/academy/herocourse/module3.html#lesson-1-one-to-many-entities